Rakefile

```diff
@@ -14,3 +14,22 @@ RuboCop::RakeTask.new(:rubocop).tap do |task|
 end
 
 task default: [:spec, :rubocop]
+
+desc "generate api models"
+task :generate_models do
+  require "k8s/model_generator"
+
+  swagger = JSON.parse(File.read('./openapi/v1.13/swagger.json'))
+  generator = K8s::ModelGenerator.new(swagger)
+  generator.each_module do |path, contents|
+    file = 'lib/' + path
```

Comment:
Maybe it should put them in a special dir so that it can clear the old ones before generating new?

Reply:
Would make sense to clear it. Still not sure if `lib/k8s/model` is a good name for these.